src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
```

Comment:
Given that you are depending on the netstandard1.7 version of System.Runtime you will also need to update the netstandard version of this library to match it.